<launch>
    <!-- Make sure you run ps3joy.py script before launching this! -->

    <!-- Start joy node to publish events to /joy topic -->
    <node name="joy_node" pkg="joy" type="joy_node" output="screen" required="true">
        <param name="dev" type="string" value="/dev/input/js0" />
        <param name="deadzone" value="0.12" />
    </node>
    
    <!-- Axes -->
    <param name="axis_linear" value="1" type="int"/>
    <param name="axis_angular" value="0" type="int"/>
    <param name="scale_linear" value="1.0" type="double"/>
    <param name="scale_angular" value="1.0" type="double"/>
    
    <!-- teleop_joy node (/joy to /cmd_vel) -->
    <node name="teleop_joy" pkg="fox_teleop" type="teleop_joy.py" output="screen" required="true"/>

</launch>